spa/src/oauth-manager/index.tsx

```diff
@@ -45,6 +47,23 @@ const OauthManager = () => {
 		return false;
 	}
 
+	async function installNewApp(onFinish: () => void) {
+		const app = await ApiRequest.gitHubApp.getAppNewInstallationUrl();
+		const exp = new Date(new Date().getTime() + FIFTEEN_MINUTES_IN_MS);
+		document.cookie = `is-spa=true; expires=${exp.toUTCString()}; path=/; SameSite=None; Secure`;
+		const winInstall = window.open(app.data.appInstallationUrl, "_blank");
+		const hdlWinInstall = setInterval(() => {
+			if (winInstall?.closed) {
+				try {
+					document.cookie = "is-spa=;expires=Thu, 01 Jan 1970 00:00:00 GMT; path=/; SameSite=None; Secure";
```

Comment:
Can't we just clear the cookie?

Reply:
Hmm, I don't know how to "clearCookie", it seems this is the way to "remove" cookie in js?